source/admin_manual/list_of_events.rst

```diff
@@ -1791,3 +1795,79 @@ happens.
 +-----------------------+------------------------------------------------------+
 | object_id             | Object ID in the storage                             |
 +-----------------------+------------------------------------------------------+
+
+Dictionaries
+============
+
+.. versionadded:: 2.3.11
+
+Events emitted by dictionary drivers and dictionary server.
+
+.. _event_dict:
+
+Common fields for dict events
+-----------------------------
+
++----------------+------------------------------------------------------+
+| Field          | Description                                          |
++================+======================================================+
+| driver         | Name of the dictionary driver                        |
++----------------+------------------------------------------------------+
+| key            | Affected key or prefix                               |
```

Comment:
Example needed. Especially showing that it can include priv/ (or was it private/?) and shared/ prefix.

Reply:
I'll have to see how to looks like.